src/views/Team.vue

```diff
@@ -227,6 +285,10 @@
                 flex-direction: column;
 
             }
+            .container1{
+                flex-direction: row;
+                flex-wrap: wrap;
+            }
```

Comment:
...and this same?

Reply:
I have resolved the error! That was by mistake